src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -138,7 +138,7 @@ def main():
                                          retry_on_timeout=True)):
         Worker(
             ['d3a'],
-            name='simulation.{}.{:%s}'.format(getpid(), now())
+            name='simulation.{}.{:%s}'.format(getpid(), now()),log_job_description=False
```

Comment:
```suggestion
            name='simulation.{}.{:%s}'.format(getpid(), now()), log_job_description=False
```
Please follow flake8 in order of travis build to pass.

Also curious to know if you used git cli to commit or a git gui tool because git cli always pre-catch it while committing.

Reply:
while commiting its not identified this actually for d3a-web its identifying, ok will remove this space

Reply:
Actually it is vice-versa: d3a should include `flake8` checks, while d3a-web does not. Having said that, what I usually do to avoid such problems is to run `tox -e travis` locally before I open a PR to make sure that my changes pass all required syntactic, unit and integration tests.

Reply:
Spyros ,Sure , I will do that